src/lib/components/support.svelte

```diff
@@ -1,120 +1,159 @@
 <script lang="ts">
     import { Button } from '$lib/elements/forms';
-    import { app } from '$lib/stores/app';
     import { wizard } from '$lib/stores/wizard';
     import SupportWizard from '$routes/(console)/supportWizard.svelte';
-    import { showSupportModal } from '$routes/(console)/wizard/support/store';
-    import { isCloud } from '$lib/system';
+    import { isSupportOnline, showSupportModal } from '$routes/(console)/wizard/support/store';
     import { trackEvent } from '$lib/actions/analytics';
-    import { localeTimezoneName, utcHourToLocaleHour } from '$lib/helpers/date';
+    import { localeShortTimezoneName, utcHourToLocaleHour } from '$lib/helpers/date';
     import { upgradeURL } from '$lib/stores/billing';
+    import { Card } from '$lib/components/index';
+    import { app } from '$lib/stores/app';
     import { currentPlan } from '$lib/stores/organization';
 
     export let show = false;
 
     $: hasPremiumSupport = $currentPlan?.premiumSupport ?? false;
 
-    $: supportTimings = `${utcHourToLocaleHour('16:00')} - ${utcHourToLocaleHour('00:00')} ${localeTimezoneName()}`;
+    $: supportTimings = `${utcHourToLocaleHour('16:00')} - ${utcHourToLocaleHour('00:00')} ${localeShortTimezoneName()}`;
+
+    type SupportOption = {
+        cta?: string;
+        icon: string;
+        label: string;
+        link?: string;
+        description: string;
+        showSupport: boolean;
+    };
+
+    const supportOptions: SupportOption[] = [
+        {
+            showSupport: true,
+            icon: 'support',
+            label: 'Premium support',
+            description: 'Get priority email support from the Appwrite team'
+        },
+
+        {
+            icon: 'discord',
+            cta: 'Discord',
+            showSupport: false,
+            label: 'Community support',
+            link: 'https://appwrite.io/discord',
+            description: 'Get support from our community through Discord'
+        },
+        {
+            icon: 'github',
+            cta: 'Open issue',
+            showSupport: false,
+            label: 'Open GitHub issue',
+            link: 'https://github.com/appwrite',
+            description: 'Report a bug or pitch a new feature'
+        }
+    ];
 </script>
 
-{#if isCloud}
```

Comment:
Some things shouldn't be shown on self-hosted, right?

Reply:
Done.